scripts/github-actions/tests-e2e-direct.sh

```diff
@@ -38,7 +38,7 @@ KCC_USE_DIRECT_RECONCILERS=LoggingLogMetric \
 GOLDEN_OBJECT_CHECKS=1 \
 GOLDEN_REQUEST_CHECKS=1 \
 E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=mock \
-  go test -test.count=1 -timeout 600s -v ./tests/e2e -run 'TestAllInSeries/fixtures/explicitlogmetric|TestAllInSeries/fixtures/exponentiallogmetric|TestAllInSeries/fixtures/linearlogmetric'
+  go test -test.count=1 -timeout 600s -v ./tests/e2e -run 'TestAllInSeries/fixtures/explicitlogmetric|TestAllInSeries/fixtures/exponentiallogmetric|TestAllInSeries/fixtures/linearlogmetric|TestAllInSeries/fixtures/monitoringdashboard'
```

Comment:
two quick points:

- this is not using the direct controller for monitoring yet
- we probably want to ignore the `s-i-s: merge`

I don't think we can do the same trick as the `script.yaml` annotations for `absent` since these are fixtures but will leave that up to you if you want to modify the yaml or add a code check (specifically for `monitoring` I think) for now.

( these are probably all items I could've done better 😛 )

Reply:
Can you share more context about the "script.yaml and the trick" (just in case others may hit a problem)?
 
I thought this go test  is a short path for LLM only: the `KCC_USE_DIRECT_RECONCILERS` specifies to LLM and the run only lists LLM related tests.
If we want to run tests for different scifi controllers, maybe we shall separate it from here?

Reply:
> I don't think we can do the same trick as the script.yaml annotations for absent since these are fixtures

Oh I see, I think this comment was a bit confusing and I should've linked the code I had in mind 🤦🏼 Here's an example of using `state-into-spec: absent` in`script.yaml`:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/86a55f99a46e5fb5a1072d6e9fa819edcfb03fb2/tests/e2e/testdata/scenarios/direct/llm_immutable_field/labels_value_type/script.yaml#L15-L20

But again, I don't think we can use that here because the `fixture` tests are shared in `test-e2e-fixtures` today, and more importantly, in post submits.

> I thought this go test is a short path for LLM only: the KCC_USE_DIRECT_RECONCILERS specifies to LLM and the run only lists LLM related tests.

Yes I meant to highlight that in with this point:

> * this is not using the direct controller for monitoring yet

So today, adding that would be a "no-op" and yes we would require a `KCC_USE_DIRECT_RECONCILERS` flag when we actually want to use the direct controller. Moreover, I'd recommend we split out the LLM and other resources 😛